getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+    "    # other params...\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "23b80bbe-442c-4374-a885-87bd6408dc28",
+   "metadata": {},
+   "source": [
+    "**AZURE_OPENAI_API_VERSION** and **AZURE_OPENAI_API_ENDPOINT** are both being passed, but **AZURE_OPENAI_API_KEY** is being retrieved within the constructor."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9b85766e-2581-4f24-86c1-cde10a0601c0",
+   "metadata": {},
+   "source": [
+    "#### Other Optional Parameters\n",
+    "\n",
+    "- `temperature` determines how creative and unpredictable, or how deterministic and predictable, the model should be in its responses. A temperature of 0 would be predictable, while anything higher would make responses more random.\n",
+    "\n",
+    "- `max_tokens` defines the maximum number of tokens (words or pieces of words) the model can generate in its response.\n",
+    "\n",
+    "- `timeout` specifies the maximum amount of time (in seconds) to wait for a response from the API before timing out.\n",
+    "\n",
+    "- `max_retries` sets the number of times the API request should be retried in case of failure before giving up.\n",
+    "\n",
+    "- `organization` is used to specify the organization ID if required for API access.\n",
+    "\n",
+    "- `model` specifies the model to be used.\n",
+    "\n",
+    "- `model_version` indicates the specific version of the chosen model to use.\n",
+    "\n",
+    "- See the [**API Reference**](https://api.python.langchain.com/en/latest/chat_models/langchain_openai.chat_models.azure.AzureChatOpenAI.html) for more details\n",
+    "\n",
+    "- Other parameters may be available in different SDK's"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28a397b3-27b7-49f0-bccf-108f39196ee6",
+   "metadata": {},
+   "source": [
+    "## Model Usage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7600d5eb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_core.messages import HumanMessage, SystemMessage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a152eab-52e5-48f3-bc8f-0b410d65c7d0",
+   "metadata": {},
+   "source": [
+    "#### Using Messages from the `langchain_core.messages` Library\n",
+    "\n",
+    "The `langchain_core.messages` library allows the user to define messages for the model and assign roles to each message."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "259a462a-4a83-4f17-a337-be04d67d3020",
+   "metadata": {},
+   "source": [
+    "- LangChain-compatible chat models take a list of `messages` as `input` and return the AI message as `output`.\n",
+    "\n",
+    "- All messages have `role` and `content` properties. However, below the roles are set using `SystemMessage` and `HumanMessage`. We'll cover more on this later.\n",
+    "\n",
+    "- Additional provider-specific information can be incorporated using the `Additional Properties`."
```

Comment:
Do we explain the Additional Properties somewhere later?

Reply:
updated to 'Additional provider-specific information can be incorporated using the `additional_kwargs` parameter. This could include provider-specific metadata or custom settings and flags.'